### ISSUE TITLE
purge all settings when open/close profile

### ISSUE THREAD

**Issue by matt-cahill** [DCS-Skunkworks/DCSFlightpanels#58]
opening a new profile without quitting causes the following error and aditional errors like Layer X cannot be found etc.
```
20.04.2020 05:36:20  version : 1.0.211.7151
1068 Custom message = []
System.NullReferenceException
Object reference not set to an instance of an object.
   at DCSFlightpanels.PanelUserControls.SwitchPanelPZ55UserControl.ClearAll(Boolean clearAlsoProfile)
   at DCSFlightpanels.PanelUserControls.SwitchPanelPZ55UserControl.SettingsCleared(Object sender, PanelEventArgs e)
```

**Comment by jdahlblom** [comment]
Check latest alpha.

**Comment by jdahlblom** [comment]
Fixed. Can you close if OK?

**Comment by matt-cahill** [comment]
The error does not appear anymore but the button UI keeps all the configuration from the old profile. The only way to get it to update is to switch to another layer and back again.

**Comment by jdahlblom** [comment]
Does it still do that? That the button grid shows which buttons had configuration (dot image) etc etc?
I can't reproduce this so I thought it had been fixed together with fixing something else.

**Comment by matt-cahill** [comment]
Yeah it's still happening for me in 4.0.39 but the buttons don't have configuration it's just that the yellow dots remain in place. e.g. here you see the button config is empty but the config indicator is still there.

![image](https://user-images.githubusercontent.com/2563358/80938865-c603d080-8e2e-11ea-9eac-371dde604c1d.png)

**Comment by jdahlblom** [comment]
This is the weirdest of them all.
After opening a profile (while a profile is already open) it is as if there are two Stream Decks objects. The button UI grid still has the first one and the rest of the controls (buttons textboxes) sees the new Stream Deck object.
I spent a long time trying to understand but did not find a solution. I implemented Dispose pattern. What is different with the button UI grid is that it is created runtime in the code, the other two (Action & Face) are defined in the XAML.
